core/src/main/java/org/bitcoinj/core/ECKey.java

```diff
@@ -705,7 +705,7 @@ public static boolean verify(byte[] data, ECDSASignature signature, byte[] pub)
 
         if (Secp256k1Context.isEnabled()) {
             try {
-                return NativeSecp256k1.verify(data, signature.encodeToDER(), pub);
```

Comment:
@greenaddress You were the last who touched native code signature verification. What do you think of these changes?